From: Michal Schmidt <mschmidt@redhat.com>
Date: Thu, 2 Sep 2010 23:23:55 -0400
Subject: [misc] drivers: remove private definitions of pr_* macros
Message-id: <20100902232354.10621.1022.stgit@brian.englab.brq.redhat.com>
Patchwork-id: 28039
O-Subject: [RHEL5.6 BZ629634 PATCH 4/5] drivers: remove private definitions of
	pr_*() helpers
Bugzilla: 629634
RH-Acked-by: David S. Miller <davem@redhat.com>
RH-Acked-by: Jiri Pirko <jpirko@redhat.com>
RH-Acked-by: Stanislaw Gruszka <sgruszka@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Remove duplicate definitions.

diff --git a/drivers/char/hvc_iucv.c b/drivers/char/hvc_iucv.c
index ace02ce..fd10131 100644
--- a/drivers/char/hvc_iucv.c
+++ b/drivers/char/hvc_iucv.c
@@ -24,12 +24,6 @@
 
 #include "hvc_console_s390.h"
 
-#ifndef pr_err
-#	define		pr_err(fmt,arg...) \
-				printk(KERN_ERR fmt,##arg)
-#endif
-
-
 /* General device driver settings */
 #define HVC_IUCV_MAGIC		0xc9e4c3e5
 #define MAX_HVC_IUCV_LINES	HVC_ALLOC_TTY_ADAPTERS
diff --git a/drivers/edac/edac_mce_amd.h b/drivers/edac/edac_mce_amd.h
index fa6ec76..e777d98 100644
--- a/drivers/edac/edac_mce_amd.h
+++ b/drivers/edac/edac_mce_amd.h
@@ -4,31 +4,6 @@
 #include <asm/mce.h>
 #include "edac_mc.h"
 
-/* backported facilities from later kernels*/
-
-#undef pr_info
-
-#ifndef pr_fmt
-#define pr_fmt(fmt) fmt
-#endif
-
-#define pr_emerg(fmt, ...) \
-       printk(KERN_EMERG pr_fmt(fmt), ##__VA_ARGS__)
-#define pr_alert(fmt, ...) \
-       printk(KERN_ALERT pr_fmt(fmt), ##__VA_ARGS__)
-#define pr_crit(fmt, ...) \
-       printk(KERN_CRIT pr_fmt(fmt), ##__VA_ARGS__)
-#define pr_err(fmt, ...) \
-       printk(KERN_ERR pr_fmt(fmt), ##__VA_ARGS__)
-#define pr_warning(fmt, ...) \
-       printk(KERN_WARNING pr_fmt(fmt), ##__VA_ARGS__)
-#define pr_notice(fmt, ...) \
-       printk(KERN_NOTICE pr_fmt(fmt), ##__VA_ARGS__)
-#define pr_info(fmt, ...) \
-       printk(KERN_INFO pr_fmt(fmt), ##__VA_ARGS__)
-#define pr_cont(fmt, ...) \
-       printk(KERN_CONT fmt, ##__VA_ARGS__)
-
 #define ERROR_CODE(x)                  ((x) & 0xffff)
 #define EXT_ERROR_CODE(x)              (((x) >> 16) & 0x1f)
 #define EXT_ERR_MSG(x)                 ext_msgs[EXT_ERROR_CODE(x)]
diff --git a/drivers/net/s2io.c b/drivers/net/s2io.c
index 675c6f9..9473938 100644
--- a/drivers/net/s2io.c
+++ b/drivers/net/s2io.c
@@ -54,9 +54,6 @@
 
 #define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
 
-#define pr_err(fmt, ...) \
-	printk(KERN_ERR pr_fmt(fmt), ##__VA_ARGS__)
-
 #include <linux/module.h>
 #include <linux/types.h>
 #include <linux/errno.h>
diff --git a/drivers/net/spider_net.h b/drivers/net/spider_net.h
index 0acf6f5..24ed9fc 100644
--- a/drivers/net/spider_net.h
+++ b/drivers/net/spider_net.h
@@ -479,7 +479,4 @@ struct spider_net_card {
 	struct spider_net_descr darray[0];
 };
 
-#define pr_err(fmt,arg...) \
-	printk(KERN_ERR fmt ,##arg)
-
 #endif
